multiaccounts/accounts/database.go

```diff
@@ -283,7 +283,7 @@ func (a *Keypair) Operability() AccountOperable {
 
 // Database sql wrapper for operations with browser objects.
 type Database struct {
-	*settings.Database
+	settings.DatabaseSettingsManager
```

Comment:
In a separate task, we can make the Database take different dependencies like Reader, Writer,... for the Database to be designed better.
I leave this out of the scope of this ticket.